# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../man-pages/inst-min.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# Copyright (C) 1998 - 2003 ROCK Linux Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---


Only install the new stuff ...

 - Clifford

--- ./Makefile.orig	2005-01-10 10:27:18.000000000 -0300
+++ ./Makefile	2005-01-10 10:27:22.000000000 -0300
@@ -17,10 +17,8 @@
 screen:
 	-mkdir not_installed
 	for i in man?/*; do \
-		if [ $(MANDIR)/"$$i" -nt "$$i" ]; then \
-			cmp -s $(MANDIR)/"$$i" "$$i" > /dev/null 2>&1; \
-			if [ "$$?" != 0 ]; then mv "$$i" not_installed; fi; \
-		fi; \
+ 		[ -f "$(MANDIR)/$$i" ] && mv -v $$i not_installed; \
+ 		true; \
 	done
 
 remove:
@@ -52,6 +50,7 @@
 	for i in man? man??; do \
 		install -d -m 755 $(MANDIR)/"$$i"; \
 		install -m 644 "$$i"/* $(MANDIR)/"$$i"; \
+		true; \
 	done; \
 	rm -f $(README) $(README).gz $(README).bz2
 
